.github/workflows/ci-tests.yml

```diff
@@ -42,6 +42,54 @@ jobs:
             source presto-product-tests/conf/product-tests-${{ matrix.config }}.sh &&
               presto-hive-hadoop2/bin/run_hive_s3_tests.sh
           fi
+      - name: Run Azure Wasb Tests
+        env:
+            WASB_CONTAINER: "${WASB_TESTS_WASB_CONTAINER}"
+            WASB_ACCOUNT: "${WASB_TESTS_WASB_ACCOUNT}"
+            WASB_ACCESS_KEY: "${WASB_TESTS_WASB_ACCESS_KEY}"
+        run: |
+            if [ "${WASB_CONTAINER}" != "" ]; then
+              presto-hive-hadoop2/bin/run_hive_wasb_tests.sh
```

Comment:
This doesn't `source presto-product-tests/conf/product-tests-${{ matrix.config }}.sh` and probably it doesn't need to.

Maybe move out of `hive-tests` job to eg `hive-tests-azure` that's not matrix.

@electrum opinion?